db/seeds/admin_login_setup.rb

```diff
@@ -0,0 +1,75 @@
+# rubocop:disable Rails/Output
```

Comment:
I used our seed file for local development and pulled out the bare minimum out of it to be able to test the admin login.

Reply:
Hmm, I think I would prefer this file to be in something like `spec/support/seeds/e2e/admin_login_setup.rb` or similar, so there can be absolutely no confusion about what this is used for. WDYT?